### PR TITLE
Aviate overview

### PR DESCRIPTION
**Pull request by sbrossie**
I am  setting this PR for review since we already started to merge some Aviate docs -e.g. [here](https://github.com/killbill/killbill-docs/pull/575). It includes:
* Bug fixes and in 5b540a36ece229c4bc5c9f27678942cf2f091c8d, 8c6abc61340f4e024b9be7c0971deffa13244293, 89b3f925f98c83ce4fa6e470d626d7f2604aac39
* Draft page for 'What is aviate' in 89b3f925f98c83ce4fa6e470d626d7f2604aac39

Page looks like:

Screen Shot 2024-11-04 at 2 40 18 PM

### CODE REVIEW

File: userguide/getting-started/what_is_aviate.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+== About Aviate
+Aviate is designed to deliver a seamless billing experience that builds upon the core capabilities of Kill Bill. Our goal is to provide users with advanced features that cover the entire quote-to-cash lifecycle, including enhanced system observability, streamlined configurations, and valuable business intelligence.
+
+== Beta Program Access
+As part of the Beta program, you'll have access to early feature releases directly on https://aviate.killbill.io[aviate.killbill.io]. This platform will continue to evolve with new functionalities aimed at making billing management easier and more powerful.
+
+== Aviate Plugin
+For Beta customers, we offer the *Aviate plugin*. This plugin extends Kill Bill with additional endpoints specifically for the Beta phase, allowing early access to some of Aviate's backend features. These endpoints are detailed in the *Aviate section* below.
+
+Please refer to the endpoint documentation below to start integrating Aviate into your Kill Bill environment.
```
Comment (reshmabidikar) [inline userguide/getting-started/what_is_aviate.adoc]: Do we need to add a link to the API docs here?

File: html5/_main_toc.html.slim [hunk 2]
```diff
@@ -20,6 +20,9 @@ nav.sidebar-nav
         li
           a.nav-link href="/latest/quick_start_with_kb_api.html"
             | Quick Start with the Kill Bill API
+        li
+          a.nav-link href="/latest/what_is_aviate.html"
+            | What is Aviate?                          
     li
```
Comment (reshmabidikar) [inline html5/_main_toc.html.slim]: Should this be moved to the new Aviate section here?

image

**Review by reshmabidikar — APPROVED** [review]
Looks good, I've left a few minor comments. Also, we will need to add a link to this doc in the other docs that I created.  I can do this is a separate PR.

File: html5/_main_toc.html.slim (hunk 2 above)
Comment (sbrossie) [reply html5/_main_toc.html.slim]: Fixed in [03c6e0c](https://github.com/killbill/killbill-docs/pull/571/commits/03c6e0c32c27053a151a4c8a73f9e923fecea4c7)

File: userguide/getting-started/what_is_aviate.adoc (hunk 1 above)
Comment (sbrossie) [reply userguide/getting-started/what_is_aviate.adoc]: Fixed in [2236cd9](https://github.com/killbill/killbill-docs/pull/571/commits/2236cd9c109b45c48f44d059fcc86d1e689de5e0). I simply referenced the "pages below"


+ Added the missing banner.

**Pull request merged**